examples/kprobetcp/kprobetcp/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use aya::{include_bytes_aligned, programs::KProbe, Bpf};
+use aya_log::BpfLogger;
+use clap::Parser;
+use log::{info, warn};
+use tokio::signal;
+
+#[derive(Debug, Parser)]
+struct Opt {}
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let _opt = Opt::parse();
+
+    env_logger::init();
+
+    // This will include your eBPF object file as raw bytes at compile-time and load it at
+    // runtime. This approach is recommended for most real-world use cases. If you would
+    // like to specify the eBPF program at runtime rather than at compile-time, you can
+    // reach for `Bpf::load_file` instead.
+    #[cfg(debug_assertions)]
+    let mut bpfh = Bpf::load(include_bytes_aligned!(
```

Comment:
Where does the `h` in `bpfh` come from? In aya-template, that variable is just called `bpf`.

Reply:
changed in 1257df4